src/obb_demo/boardground.cljs

```diff
@@ -224,13 +224,15 @@
 (defn register-action
   "Registers an action"
   [game-data game player action coord]
-  (let [current-actions (:actions game-data)
+  (let [current-actions (or (:actions game-data) [])
+        next-actions (conj current-actions action  )
```

Comment:
I guess you forgot to clean up the trailing whitespaces before the parentheses here :p